.jshintrc

```diff
@@ -0,0 +1 @@
+{ "esversion": 6 }
```

Comment:
Add this to your .gitignore. If Ian and I are not using jshint, we don't need this file.